tasks/install-on-Darwin.yml

```diff
@@ -13,13 +13,21 @@
     state: latest
     update_homebrew: yes
 
+- name: Configure Buildkite service
+  template:
+    src: "homebrew.mxcl.buildkite-agent.plist.j2"
+    dest: "{{ buildkite_agent_conf_dir[ansible_os_family] }}/homebrew.mxcl.buildkite-agent.plist"
+    mode: "0600"
```

Comment:
Please would you make explicit the filesystem owner and group? I think the group can probably be `staff` or `admin` on macOS.